index.html

```diff
@@ -38,197 +51,234 @@ <h1 class="hero-title">
         </div>
-    </section>
-
-    <section class="careers-card-5">
-        <div class="careers-card">
-        <img src="Images/img-placeholder.png" alt="placeholder">
+        </div>
+    </div>
+    <div class="careers-card">
+        <div class="img-Placeholder"></div>
         <div class="careers-card-footer">
-            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
-            <ul class="tags">
+        <h2 id="careers_title">Multi-Post Stories Gain+Glory</h2>
+        <div class="languages-category">
+            <ul id="career-tags">
                 <li class="Lang-tag">Ruby on rails</li>
                 <li class="Lang-tag">Css</li>
                 <li class="Lang-tag">Javascript</li>
                 <li class="Lang-tag">Html</li>
             </ul> 
-            <div class="action">
-                <button class="btn-primary primary-font-style">See project</button>
-            </div>
         </div>
+        <div id="action">
+            <button type="button" class="project-button">See Project</button>
         </div>
-    </section>
-
-    <section class="careers-card-6">
-        <div class="careers-card">
-        <img src="Images/img-placeholder.png" alt="placeholder">
+        </div>
+    </div>
+    <div class="careers-card">
+        <div class="img-Placeholder"></div>
         <div class="careers-card-footer">
-            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
-            <ul class="tags">
+        <h2 id="careers_title">Multi-Post Stories Gain+Glory</h2>
+        <div class="languages-category">
+            <ul id="career-tags">
                 <li class="Lang-tag">Ruby on rails</li>
                 <li class="Lang-tag">Css</li>
                 <li class="Lang-tag">Javascript</li>
                 <li class="Lang-tag">Html</li>
             </ul> 
-            <div class="action">
-                <button class="btn-primary primary-font-style">See project</button>
-            </div>
+        </div>
+        <div id="action">
+            <button type="button" class="project-button">See Project</button>
         </div>
         </div>
-    </section>
     </div>
-</section>
+    </div>
+    </section>
 
 
     <!-- About me section -->
 
     
-    <section id="about_me" class="center-content">
-        <h1 class="primary-font-style primary-font-size"><b>About me</b></h1>
-        <p class="supporting_text primary-font-style">
-             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
-        </p>
-        <button class="btn-primary primary-font-style btn-resume">Get My Resume</button>
-    </section>
+    <section class="about_me_section"><div class="about-me">
+        <div class="about-me-title"><h2 id="about-me-text">About me</h2>
+        <div class="about">
+            <img src="images/About.png" alt="About">
```

Comment:
- [ ] Kindly note that all images meant for decoration purposes should be added as `background-image` in your CSS file. Kindly move these images to your CSS file. This is best practice. Kindly check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more details.